src/window-behavior.js

```diff
@@ -255,12 +290,14 @@ export default class WindowBehavior {
       if (resize) {
         this.wasResized = true;
         win._setState('resizing', true, false);
-        win.setDimension(getNewDimensions(
+        win.resizeWindow(getNewPositionsAndDimensions(
+          target.getAttribute('class').substring(19),
```

Comment:
This feels a bit strange. If the idea is to get the last part of the string of the className you might be better of using a custom data- attribute and just straight up get the value that way. This will break if the class attribute is modified.